smart-contracts/package.json

```diff
@@ -59,7 +59,8 @@
     "unlock-abi-1-0": "1.0.0",
     "unlock-abi-1-1": "1.1.1",
     "unlock-abi-1-2": "1.0.4",
-    "unlock-abi-1-3": "1.0.2"
+    "unlock-abi-1-3": "1.0.2",
+    "unlock-abi-7": "^1.0.2"
```

Comment:
Let's pin the dependency like the others

Reply:
yup. will do.